pkgs/applications/video/mediathekview/default.nix

```diff
@@ -37,6 +38,18 @@ stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
+  desktopItems = makeDesktopItem {
+    name = "MediathekView";
+    exec = meta.mainProgram;
+    icon = "MediathekView";
+    desktopName = meta.mainProgram;
+    genericName = "MediathekView";
+    comment = meta.description;
```

Comment:
A change to meta should ideally not trigger a rebuild

Reply:
Good call. I just set those attributes directly now, without referring to `meta`.